snapcraft/plugins/v1/catkin.py

```diff
@@ -68,6 +68,10 @@
       (string)
       The URI to ros master setting the env variable ROS_MASTER_URI. Defaults
       to http://localhost:11311.
+    - skip-keys:
+      (list of strings)
+      List of catkin packages to skip when resolving dependencies. This is useful
+      when a package has not been added to rosdep but is known to be installed.
```

Comment:
I'm not sure "installed" is the right word in the context of building the snap. Maybe "staged"?

Reply:
fixed.